ProjectLighthouse/Configuration/ConfigurationCategories/CustomizationConfiguration.cs

```diff
@@ -3,4 +3,6 @@ namespace LBPUnion.ProjectLighthouse.Configuration.ConfigurationCategories;
 public class CustomizationConfiguration
 {
     public string ServerName { get; set; } = "Project Lighthouse";
+    public string EnvironmentName { get; set; } = "lighthouse";
+    public bool UseNumericRevisionNumber { get; set; } = false;
```

Comment:
Using the revision count is never going to be reliable; shallow clones exist and are used frequently with CI systems (both GitHub Actions and Gitlab CI only clone up to 20 commits)

I once again strongly against using revision counts; hashes are better in every way. If readability is a problem, it may be worth trimming the number of characters from 8 to about 6.

Reply:
issues like this are entirely why it's optional and it's just for preference if you can facilitate the use of numeric revisions

I guess I can add a disclaimer in the envver that it's built from a shallow clone, or disable the functionality and throw a warning in the console that shallow clone builds don't support numeric versioning?

Reply:
It's better and easier to just not have the option due to feature creep/bloat.

You're welcome to keep this functionality in a fork if you really want it, but it doesn't belong in the main repo.

Reply:
it's kinda a balancing act between the end user being able to tell easier that the server's been updated, and reliability.

Reply:
That is what a versioning system like SemVer (which is also a bit more clear) can provide. It will be added when Lighthouse gets closer to release.

I'm going to be brutally honest; this is the hill I will die on. Please remove this.

Reply:
I will once I finish testing other changes as I personally use this.